### PR TITLE
DB Replica database_tasks as ENV

### PR DESCRIPTION
**Pull request by shleeable**
Investigating ReadySet as a read-only replica cache.

Reference: 

- https://readyset.io/docs/get-started/connect/connect-an-application-via-an-orm/ruby-on-rails-with-readyset
- https://guides.rubyonrails.org/active_record_multiple_databases.html

### CODE REVIEW

**Comment by shleeable** [comment]
Created the Doco PR https://github.com/mastodon/documentation/pull/1561

**Comment by ClearlyClaire** [comment]
Could you elaborate on this change?

File: config/database.yml [hunk 1]
```diff
@@ -60,5 +60,6 @@ production:
     password: <%= (ENV['REPLICA_DB_PASS'] || ENV['DB_PASS'] || '').to_json %>
     host: <%= ENV['REPLICA_DB_HOST'] ||ENV['DB_HOST'] || 'localhost' %>
     port: <%= ENV['REPLICA_DB_PORT'] ||ENV['DB_PORT'] || 5432 %>
-    prepared_statements: <%= ENV['PREPARED_STATEMENTS'] || 'true' %>
+    prepared_statements: <%= ENV['REPLICA_PREPARED_STATEMENTS'] ||ENV['PREPARED_STATEMENTS'] || 'true' %>
```
Comment (ClearlyClaire) [inline config/database.yml:63]: Minor style issue, not sure why it isn't caught by anything:
```suggestion
    prepared_statements: <%= ENV['REPLICA_PREPARED_STATEMENTS'] || ENV['PREPARED_STATEMENTS'] || 'true' %>
```

Comment (shleeable) [reply config/database.yml:63]: Do you want me to add the space for all of the ORs?

**Comment by shleeable** [comment]
> Could you elaborate on this change?

The `database_tasks` being false is required for ReadySet.

Per documentation.  `It's important to note that the primary_replica definition points to Readyset and that two flags are set: replica is true and database_tasks is false.`

and the optional replica `prepared_statements` is for safety (I have no confirmed or denied if ReadySet supports prepared statements).

**Comment by shleeable** [comment]
Update: I can remove the optional prepared_statement change, but it also doesn't hurt.

ref https://readyset.io/docs/reference/features/queries

**Review by ClearlyClaire — APPROVED** [review]

**Pull request merged**